deploy/webhook.yaml

```diff
@@ -235,10 +235,12 @@ spec:
         - name: webhook
```

Comment:
Can you revert the changes to this file? `git checkout origin/master deploy/webhook.yaml`